includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -2005,6 +2005,15 @@ private function ampify_ruleset_selectors( $ruleset ) {
 				continue;
 			}
 
+			/*
+			 * Remove any rule with a responsive styling class for blocks, which isn't needed in AMP.
+			 * @link https://github.com/WordPress/gutenberg/blob/9a16ac09ddff4b0bf12430d1426c4cfefa75b56d/packages/block-library/src/embed/style.scss#L26
+			 */
+			if ( preg_match( '/wp-embed-aspect-\S/', $selector ) ) {
```

Comment:
This works, but I was thinking that the underlying class name could be removed from the block HTML element itself. The CSS would then be stripped as a matter of course by the tree shaker.